frontend/nc-games/src/App.js

```diff
@@ -1,23 +1,21 @@
 import './App.css';
-
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Welcome from "./components/Welcome";
+import Reviewlist from "./components/Reviewlist";
+import Categories from "./components/Categories";
 function App() {
+  
+
   return (
+    <BrowserRouter>
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+          <Routes>
+            <Route path="/Welcome" element={<Welcome />} />
```

Comment:
I'd probably change the path here to just "/". At the moment, people have to come to an endpoint of "/Welcome" in order to see anything on your site, but if you think about how websites traditionally work, you'd expect to see something even just on a blank endpoint (you would expect to see something if you went to bbc.co.uk, not have to manually go to bbc.co.uk/welcome)